webapp/app/[locale]/stake/_components/manageStake/website.tsx

```diff
@@ -0,0 +1,42 @@
+import { ExternalLink } from 'components/externalLink'
+import { ArrowDownLeftIcon } from 'components/icons/arrowDownLeftIcon'
+import { useTranslations } from 'next-intl'
+import { StakeToken } from 'types/stake'
+
+const websitesMap: Partial<
+  Record<StakeToken['extensions']['protocol'], string>
+> = {
+  bedRock: 'https://www.bedrock.technology',
+  bitFi: 'https://www.bitfi.one',
+  exSat: 'https://exsat.network',
+  lorenzo: 'https://lorenzo-protocol.xyz',
+  merlinChain: 'https://merlinchain.io',
+  nodeDao: 'https://www.nodedao.com',
+  pumpBtc: 'https://mainnet.pumpbtc.xyz',
+  solv: 'https://solv.finance',
+  stakeStone: 'https://stakestone.io/#/home',
+  uniRouter: 'https://www.unirouter.io/#',
```

Comment:
Perhaps we should move this into the token-list too. Having part of the information there and part here seems an invitation to mismatches, missing data, etc.

Reply:
I am unsure if we want to bloat the token list with this informational stuff, which is part of the portal. We also need to add the reward list mapped to each tokens. Would you prefer to move that into the token list? I'd rather not 🤔

Reply:
As discussed offline, I convinced him and we're keeping this info in the portal, and keeping the token-list as is